Mac/MainWindow/MainWindowController.swift

```diff
@@ -273,6 +273,11 @@ class MainWindowController : NSWindowController, NSUserInterfaceValidations {
 		openArticleInBrowser(sender)
 	}
 
+	@objc func openFeedInAppBrowser(_ sender: Any?) {
```

Comment:
Could we change this to `openInAppBrowser`?  It isn't really opening the feed, it is opening the article.

Reply:
Sure it was conflicting between iOS [return] key. I will make the changes everywhere

Reply:
I made changes for iOS and found for Mac feed list was not opening in browser fixed that as well.

So for Mac and iOS - both feed list and articles list - both `b` and `[return]` key works as expected.